Packages/Core/Editor/Drawers/AtomDrawer.cs

```diff
@@ -1,4 +1,3 @@
-#if UNITY_2019_1_OR_NEWER
```

Comment:
should be 2018_4_or_newer
instead of removing, or?

Reply:
updated to have UNITY_2018.4_OR_NEWER and also updated the summary comment.